Dockerfile

```diff
@@ -1,15 +1,5 @@
-FROM ubuntu:12.04
+FROM python:3.4-alpine
 
-RUN apt-get update
-RUN apt-get install -y wget ca-certificates
-
-# Install pip
-RUN cd /tmp; wget https://raw.github.com/pypa/pip/master/contrib/get-pip.py;\
- python get-pip.py; rm get-pip.py;
-
-# Install pip via package-manager 
-RUN apt-get install -y python-pip
-
-# add flower
-RUN pip install redis flower
+ENV FLOWER_VERSION 0.9.1
```

Comment:
`pip install -U flower` installs the latest version. No need to specify the version.